android/build.gradle

```diff
@@ -38,8 +38,8 @@ android {
 }
 
 dependencies {
-    implementation 'androidx.work:work-runtime:2.5.0'
+    implementation 'androidx.work:work-runtime:2.7.0-beta01'
```

Comment:
This one is needed because internally WorkManager was also using `PendingIntent` without the required flag(s), which was fixed in alpha-02: https://developer.android.com/jetpack/androidx/releases/work#2.7.0-alpha02